content/hi/docs/setup/production-environment/windows/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: "कुबेरनेट्स में विंडोज़"
```

Comment:
```suggestion
title: कुबेरनेट्स में Windows
```

Windows is a trademark of Microsoft _(i think)_.

Reply:
Oh yeah! sorry, I didn't thought about it